.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# see:
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners
```

Comment:
I don't think this belongs with sig-core. It's not controller/router/cli or a library supporting one of those things.